app/services/revocation_registry.py

```diff
@@ -309,42 +311,137 @@ async def revoke_credential(
             )
         )
     except ApiException as e:
-        bound_logger.info(
+        bound_logger.exception(
             "An ApiException was caught while revoking credential. The error message is: '{}'.",
             e.reason,
         )
         raise CloudApiException(
             f"Failed to revoke credential: {e.reason}.", e.status
         ) from e
 
-    if not auto_publish_to_ledger:
-        try:
-            rev_reg_record = await get_active_revocation_registry_for_credential(
-                controller=controller,
-                credential_definition_id=credential_definition_id,
-            )
+    bound_logger.info("Successfully revoked credential.")
 
-            await publish_revocation_entry_to_ledger(
-                controller=controller,
-                revocation_registry_id=rev_reg_record.revoc_reg_id,
-                create_transaction_for_endorser=True,
-            )
-        except CloudApiException as e:
-            if e.status_code == 400:
-                bound_logger.info(
-                    "Bad request: Cannot publish revocation entry to ledger: {}",
-                    e.detail,
-                )
-            else:
-                bound_logger.error(e.detail)
-            raise e
-        except Exception as e:
-            bound_logger.exception("Exception caught when revoking credential.")
-            raise e
 
-        await endorser_revoke()
```

Comment:
This means the `endorser_revoke` function is never used and can be deleted as well.

Note that the unit tests in `app/tests/services/test_revocation_registry.py` will probably have to be edited to account for this